### PR TITLE
Update configure-email-forwarding.md

### PR DESCRIPTION
**Pull request by msbemba**
Added steps to manage remote domains because When users configure automatic forwarding to recipients on a remote domain, the remote domain settings override users' automatic forwarding settings (messages are blocked if automatic forwards are disabled for the remote domain).

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5053

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/email/configure-email-forwarding.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ As the admin of an organization, you might have company requirements to set up e
 
 Before you set up email forwarding, note the following:
 
+- Allow automatic forwarded messages to be sent to people on the remote domain. [Manage remote domains](/exchange/mail-flow-best-practices/remote-domains/manage-remote-domains)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/configure-email-forwarding.md]:
```suggestion
- Allow automatically forwarded messages to be sent to people on the remote domain. See [Manage remote domains](/exchange/mail-flow-best-practices/remote-domains/manage-remote-domains) for details.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this Pr. Thanks.

**Pull request merged**